.travis.yml

```diff
@@ -10,20 +10,21 @@ sudo: false
 
 cache:
   directories:
+    - .phpunit
     - $HOME/.composer/cache/files
 
 env:
-  matrix: SYMFONY_VERSION=2.8.*
-  global: SYMFONY_DEPRECATIONS_HELPER=321
+  matrix: SYMFONY_VERSION=3.2.*
+  global: SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
 
 matrix:
   include:
-    - php: 7.0
-      env: DEPS=dev SYMFONY_VERSION=3.1.*
+    - php: 7.1
+      env: DEPS=dev SYMFONY_VERSION=3.3.*
     - php: 5.5
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
-      env: DEPS=dev COMPOSER_FLAGS="--prefer-stable" SYMFONY_VERSION=3.0.*
```

Comment:
i would prefer to keep building with 3.0 for now. if we find reasons to want to drop 3.0 support we can still do that. allowing it without testing seems wrong. if you don't agree, lets take this change out of this PR so we can merge it.